packages/opentelemetry/test/integration/scope.test.ts

```diff
@@ -111,6 +120,7 @@ describe('Integration | Scope', () => {
               tag1: 'val1',
               tag2: 'val2',
               tag3: 'val3',
+              tag4: 'val4',
```

Comment:
this aligns the behavior with the core one, as `Sentry.setTag()` sets on the isolation scope which will be applied to the transaction too.